.github/workflows/test.yml

```diff
@@ -36,10 +36,19 @@ jobs:
             ${{ env.CONDA_ENV_PATH }}
           key: ${{ env.RUN_STAMP }}-conda-${{ hashFiles('omniscidb/scripts/mapd-deps-conda-dev-env.yml') }}-${{ inputs.cache-suffix }}
 
-      - name: Check style
+      - name: C++ style check
         if: inputs.test == 'style'
         run: |
           conda run --no-capture-output -n omnisci-dev bash -c "find . -name python -prune -or -name build -prune -or -name ThirdParty -prune -or -name third_party -prune -or \( -name '*.cpp' -print -or -name '*.h' -print \) | xargs clang-format --dry-run --Werror -style=file"
+        shell: bash
+
+      - name: Py style check
+        if: inputs.test == 'style'
+        run: |
+          source $CONDA/etc/profile.d/conda.sh
```

Comment:
I don't know if this conda activation approach would work in every case. When conda is installed with default settings, the recommended activation script is:
```
eval "$($PATH_TO_CONDA_ROOT/bin/conda shell.$YOUR_SHELL hook)"
```
Not having to escape characters in complex commands is nice though.